doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -38,7 +39,8 @@ Make sure that the version number is the same as the `Uno.UI` package.
     - Resolution: Restart VS and rebuild the app
 - The reload server may start twice (The VS **Uno Platform** output window shows two "Starting server" messages)
     - Resolution: Restart visual studio, rebuild the app.
-- The app does not update its XAML, because the port number in `HotReload.g.cs` is `0`.
+- The app does not update its XAML, because the port number in `RemoteControl.g.cs` is `0`.
+    - Ensure you have the right version of the _VSIX_ installed.
```

Comment:
```suggestion
    - Ensure you have the latest version of the _VSIX_ installed.
```

Right is not very useful as we don't say which one :)

Reply:
@jeromelaban mergify was able to merge the even though there is still an unresolved comment.

Reply:
Feels like a race condition :)